webapp/src/components/Minimap.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import './Minimap.css'
+
+const PARCELS = 500
```

Comment:
Both this and the `250` below are related to the max number of parcels? if so, `parcelUtils` has a `getBounds()` method used by the rest of the codebase to attest how many parcels are there.

What I mean is that I'd extract it **if** this two numbers are related to some other part of the codebase to simplify future changes